tests/Tests/400_ods_dashboard/400_ods_dashboard.robot

```diff
@@ -1,41 +1,46 @@
 *** Settings ***
 Resource         ../../Resources/ODS.robot
-Resource         ../../Resources/Common.robot
-Library         RequestsLibrary
+Resource        ../../Resources/Page/ODH/ODHDashboard/ODHDashboard.resource
 Test Setup      Dashboard Test Setup
 Test Teardown   Dashboard Test Teardown
 
+
 *** Keywords ***
 Dashboard Test Setup
   Set Library Search Order  SeleniumLibrary
+  Open Browser  ${ODH_DASHBOARD_URL}  browser=${BROWSER.NAME}  options=${BROWSER.OPTIONS}
+  Login To RHODS Dashboard  ${TEST_USER.USERNAME}  ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
+  Wait for RHODS Dashboard to Load
 
 Dashboard Test Teardown
   Close All Browsers
 
```

Comment:
i would suggest to move the keyword section after Test case to maintain the consistency across the script

Reply:
done